composer.lock

```diff
@@ -2423,68 +2974,59 @@
             ],
             "description": "Internal MaxMind Web Service API",
             "homepage": "https://github.com/maxmind/web-service-common-php",
-            "time": "2019-12-12T15:56:05+00:00"
+            "support": {
+                "issues": "https://github.com/maxmind/web-service-common-php/issues",
+                "source": "https://github.com/maxmind/web-service-common-php/tree/v0.8.1"
+            },
+            "time": "2020-11-02T17:00:53+00:00"
         },
         {
             "name": "monolog/monolog",
-            "version": "2.3.4",
+            "version": "1.26.1",
```

Comment:
The Symfony monolog bundle was downgraded from 5.x to 4.4, which is what we want. But a side effect of this is a major version downgrade for monolog. We should check to make sure this doesn't have any unintended side effects.